packages/connect-webextension/README.md

```diff
@@ -54,6 +54,10 @@ Amend your manifest.json to include the script as a content script. Replace <pat
   ],
 ```
 
+## Adding your webextension to `knownHosts`
+
+If you want your extension to be display with the name of the webextension instead of the webextension ID you have to open a Pull Request to get it included in the `knownHosts` in the file https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/data/config.ts#L17.
```

Comment:
Although the changes required for this are in connect-package I think this information is more useful here. The wording can probably be improve. :man_shrugging: